awx/main/management/commands/host_metric.py

```diff
@@ -0,0 +1,66 @@
+from django.core.management.base import BaseCommand
+from dateutil import parser
+from datetime import timezone
+from awx.main.models.inventory import HostMetric
+import json
+
+
+class Command(BaseCommand):
+
+    help = 'This is for offline licensing usage'
+
+    def add_arguments(self, parser):
+        parser.add_argument('--since', dest='since', action='store', help='Start date for data')
+        parser.add_argument('--until', dest='until', action='store', help='End date for data')
+        parser.add_argument('--json', action='store_true', help='Select output as JSON')
+
+    def handle(self, *args, **options):
+        opt_since = options.get('since')
+        opt_until = options.get('until')
+
+        # user gives no arguments
+        if opt_since is None and opt_until is None:
+            print("No Arguments received")
+            return None
+
+        since = None
+        if opt_since:
+            since = parser.parse(opt_since)
```

Comment:
This current approach *probably* throws an exception if you pass in a string that doesn't look like a date.  By using the approach described above, you'll should get error messages like the one I described for free.